stardog-userrole-operator/test/clusterrole_test.go

```diff
@@ -0,0 +1,65 @@
+package test
```

Comment:
The duplicated strings should be defined as constants (expected and error messages). (Applies to all *_test.go)

Reply:
I believe in duplicating the code in unit tests for better readability.